src/test/java/org/rulez/demokracia/pdengine/authentication/AuthenticatedUserServiceTest.java

```diff
@@ -0,0 +1,34 @@
+package org.rulez.demokracia.pdengine.authentication;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.rulez.demokracia.pdengine.testhelpers.ThrowableTester;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AuthenticatedUserServiceTest extends ThrowableTester {
```

Comment:
I don't see the test feature/operation/behaviour annotations. This is true to other test classes also.